packages/repo-metadata-lint/src/validate.ts

```diff
@@ -99,6 +100,17 @@ export class Validate {
       }
     }
 
+    // On success, store an entry in a metadata table. This data is
+    // used to answer questions such as API coverage:
+    if (result.status === 'success') {
+      await StoreMetadata.storeMetadata({
+        release_level: repoMetadata.release_level,
+        language: repoMetadata.language,
+        repository: repoMetadata.repository,
+        api_service: `${repoMetadata.api_shortname}.googleapis.com`,
+      });
```

Comment:
Consider also storing the raw data as a string in a field. BQ has helpers to parse JSON and read fields to avoid the JSON not having a strict schema